geode-docs/basic_config/data_entries_custom_classes/using_custom_classes.html.md.erb

```diff
@@ -49,3 +49,10 @@ The region uses hashing on keys. If you define a custom class to use as a key, f
 -   `equals`
 -   `hashCode`. The default `hashCode` inherited from `Object` uses identity, which is different in every system member. In partitioned regions, hashing based on identity puts data in the wrong place. For details, see the Java API documentation for `java.lang.Object`.
 
+Do not base the `hashCode` implementation on an enumerated type
+(`enum`) data member within a key.
```

Comment:
Maybe this should just say - do not *call* `hashCode` on an `Enum` in your `hashCode` method? You can *use* an enum, but you'll have to come up with your own way of hashing it.